test_scripts/Smoke/API/001_SetGlobalProperties_PositiveCase_SUCCESS.lua

```diff
@@ -0,0 +1,145 @@
+-- SDL checks if TTS interface is available on HMI
+-- SDL checks if SetGlobalproperties is allowed by Policies
+-- SDL checks if all parameters is allowed by Policies
+-- SDL transfers the UI part of request with allowed parameters to HMI
+-- SDL transfers the TTS part of request with allowed parameters to HMI
+-- SDL receives UI and TTS part of response from HMI with "SUCCESS" result code
+-- SDL transfers response to mobile app
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmokeApi = require('test_scripts/Smoke/commonSmokeApi')
+
+--[[ Local Variables ]]
+local putFileParams = {
+	requestParams = {
+	    syncFileName = 'icon.png',
+	    fileType = "GRAPHIC_PNG",
+	    persistentFile = false,
+	    systemFile = false
+	},
+	filePath = "files/icon.png"
+}
+
+local requestParams = {
+	helpPrompt = {
+		{
+			text = "Help prompt",
+			type = "TEXT"
+		}
+	},
+	timeoutPrompt =	{
+		{
+			text = "Timeout prompt",
+			type = "TEXT"
+		}
+	},
+	vrHelpTitle = "VR help title",
+	vrHelp = {
+		{
+			position = 1,
+			image = {
+				value = "icon.png",
+				imageType = "DYNAMIC"
+			},
+			text = "VR help item"
+		}
+	},
+	menuTitle = "Menu Title",
+	menuIcon = {
+		value = "icon.png",
+		imageType = "DYNAMIC"
+	},
+	keyboardProperties = {
+		keyboardLayout = "QWERTY",
+		keypressMode = "SINGLE_KEYPRESS",
+		limitedCharacterList = {"a"},
+		language = "EN-US",
+		autoCompleteText = "Daemon, Freedom"
+	}
+}
+
+local responseUiParams = {
+	vrHelpTitle = requestParams.vrHelpTitle,
+	vrHelp = requestParams.vrHelp,
+	menuTitle = requestParams.menuTitle,
+	menuIcon = requestParams.menuIcon,
+	keyboardProperties = requestParams.keyboardProperties
+}
+
+local responseTtsParams = {
+	timeoutPrompt = requestParams.timeoutPrompt,
+	helpPrompt = requestParams.helpPrompt
+}
+
+local allParams = {
+	requestParams = requestParams,
+	responseUiParams = responseUiParams,
+	responseTtsParams = responseTtsParams
+}
+
+--[[ Local Functions ]]
+local function setGlobalProperties(params, self)
+	local cid = self.mobileSession1:SendRPC("SetGlobalProperties", params.requestParams)
+
+	params.responseUiParams.vrHelp[1].image.value = commonSmokeApi.getPathToSDL()
+		.. "storage/" .. commonSmokeApi.getMobileAppId() .. "_"
+		.. commonSmokeApi.getDeviceMAC() .. "/icon.png"
+	params.responseUiParams.menuIcon.value = params.responseUiParams.vrHelp[1].image.value
+
+	EXPECT_HMICALL("UI.SetGlobalProperties", params.responseUiParams)
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+
+	EXPECT_HMICALL("TTS.SetGlobalProperties", params.responseTtsParams)
+	:Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+
+	self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+
```

Comment:
@AKalinich-Luxoft Redundant empty line

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1697/commits/1f9a96a07f40b1051bd1cd0e5a3b77a979c40def